sentinel2-processing/prepare_10m_sentinel_imagery.py

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+
+"""
+This is a Python version of a bash script written by Rebecca Clarke https://gist.github.com/rebclarke/b9e55a3558be3ec76248c78379b67366
+"""
+
+import os
+import glob
+import shutil
+from osgeo import gdal
+import subprocess
+
+
+# user definated variables
+TILE_INDEX = "nz-150k-tile-index.shp"
+MIN = "800"
+MAX = "3000"
```

Comment:
These are passed to `gdal_translate -scale`; based on [its documentation](https://gdal.org/programs/gdal_translate.html#cmdoption-gdal_translate-scale), would it be useful to name them something like `MIN_SCALE` and `MAX_SCALE`?